.github/workflows/android.yml

```
@@ -122,36 +122,36 @@ jobs:
+          # Use the extracted version string as the Release Name
           name: ${{ steps.artifacts.outputs.release_name }}
           body: |
             Automatic CI build based on commit: ${{ github.sha }}
             Commit message: ${{ github.event.head_commit.message }}
           prerelease: false
           files: ${{ env.APK_FILE_V8A }}
 
-      # 第 7 步：预处理提交信息，清除 Markdown 危险字符并保留换行
+      # Step 7: preprocess the commit message, removing dangerous Markdown characters and keeping line breaks
       - name: Sanitize multiline commit message for Telegram
         run: |
```

actionlint (1.7.12)

[error] "github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)